graylog2-server/src/main/java/org/graylog2/rest/models/streams/alerts/AlertConditionSummary.java

```diff
@@ -60,6 +65,17 @@ public static AlertConditionSummary create(@JsonProperty("id") String id,
                                                @JsonProperty("parameters") Map<String, Object> parameters,
                                                @JsonProperty("in_grace") Boolean inGrace,
                                                @JsonProperty("title") @Nullable String title) {
+        checkNotNull(inGrace);
```

Comment:
No, it is used by the `(Stream)AlertConditionsResource`, where we want that flag to be included to show it for stream/alert condition details.